src/Components/Server/src/BlazorPack/NonDefaultHubProtocol.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Server.BlazorPack
+{
+    // Tells SignalR not to add the IHubProtocol with this attribute to all hubs by default
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
```

Comment:
Why inherited? We're only using it on an internal sealed class anyway, and it takes away some flexibility.

Reply:
It also doesn't really matter since we sniff by name :).